FluidNC/src/Spindles/HBridgeSpindle.cpp

```diff
@@ -0,0 +1,195 @@
+// Copyright (c) 2022 -	Santiago Palomino
+// Use of this source code is governed by a GPLv3 license that can be found in the LICENSE file.
+
+/*
+    This is a full featured TTL PWM spindle This does not include speed/power
+    compensation. Use the Laser class for that.
+*/
+#include "HBridgeSpindle.h"
+
```

Comment:
I see. The file names in that directory contain the Spindle subfix, but not the classes. 

I do not have strong arguments on this, only that an HBridge is a very generic name. HBridges are also used internally in the stepper controllers. The company I work for enforces naming class files with the complete class name, to simplify finding things. But I can rename the class as part of the next commit.

Reply:
Just be consistent with the existing files.  I understand the value of making the file name match the class name, but if we change that, we should change it everywhere, instead of having some one way and some the other.

Reply:
No problem, implementing as suggested.